code/modules/projectiles/projectile/magic.dm

```diff
@@ -27,7 +27,11 @@
 /obj/item/projectile/magic/death/on_hit(var/mob/living/carbon/G)
 	. = ..()
 	if(isliving(G))
-		G.adjustFireLoss(3000)
+		if(istype(G, /mob/living/carbon/human/machine)) //speshul snowfleks deserv speshul treetment
```

Comment:
not the proper way to check for this; you'll want to check for their species to be machines instead of their strict mob type; the IPCs running around on the map are all just plain old humans; this path only exists for spawning purposes.